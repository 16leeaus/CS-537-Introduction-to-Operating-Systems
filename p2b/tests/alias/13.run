./mysh tests/13.in | sort --ignore-case
