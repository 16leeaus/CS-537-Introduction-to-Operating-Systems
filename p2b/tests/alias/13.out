a1
a1
a1 /bin/ls tests/13.desc
a2
a2
a2 /bin/cat tests/13.desc
a3
a3
a3 /bin/mkdir tests-out/tmp
a4
a4
a4 /bin/rm -rf tests-out/tmp
a5
a5
a5 /usr/bin/touch tests/13.desc
alias
alias a1 /bin/ls tests/13.desc
alias a2 /bin/cat tests/13.desc
alias a3 /bin/mkdir tests-out/tmp
alias a4 /bin/rm -rf tests-out/tmp
alias a5 /usr/bin/touch tests/13.desc
Creates multiple aliases and runs them, then removes those aliases and tries to run them
tests/13.desc
unalias a1
unalias a2
unalias a3
unalias a4
unalias a5
